src/compiler/Stage/InferTypes/GenerateEquations.elm

```diff
@@ -297,19 +305,19 @@ generateEquations idSource ( expr, type_ ) =
             )
 
 
-findArgumentUsages : VarName -> Typed.Expr -> List Typed.Expr
-findArgumentUsages argument bodyExpr =
-    bodyExpr
+findArgumentUsages : VarName -> Typed.LocatedExpr -> List Typed.LocatedExpr
+findArgumentUsages argument located =
+    located
```

Comment:
I think the previous naming here was more informative. I'd keep `bodyExpr` there.